cloudprem/eks.tf

```diff
@@ -119,7 +119,7 @@ module "eks_cluster" {
 
   # EKS cofigurations
   cluster_name    = local.identifier
-  cluster_version = "1.18"
+  cluster_version = "1.19"
```

Comment:
Sanity check...

Do we know that EKS v1.19 is available for use in all regions? Like govcloud?

Reply:
I did not check "all" regions but it definitely works in govcloud us-gov-east-1 (which is the only govcloud region we support anyway due to the lack of code pipeline in west)